src/components/About/AboutHeader/AboutHeader.js

```diff
@@ -1,56 +1,58 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import Carousel from 'react-bootstrap/Carousel'
-import HeaderImg1 from 'images/about/about1.png';
-import HeaderImg2 from 'images/about/about2.jpg';
-import './AboutHeader.css';
+import HeaderImg1 from '../../../images/about/about1.png'
```

Comment:
Please delete the relative path of images. That is already fixed using the env variables. See README to enable them
```suggestion
import HeaderImg1 from 'images/about/about1.png'
```